plugins/woocommerce/src/Gateways/PayPal/Constants.php

```diff
@@ -143,4 +143,67 @@ class Constants {
 	 * @var string
 	 */
 	const PAYPAL_ISSUE_PAYEE_ACCOUNT_RESTRICTED = 'PAYEE_ACCOUNT_RESTRICTED';
+
+	/**
+	 * Meta key for storing PayPal payment status in order meta.
+	 *
+	 * @var string
+	 *
+	 * @since 10.5.0
+	 */
+	const PAYPAL_META_STATUS = '_paypal_status';
```

Comment:
Thought: given that all of these values are order meta fields, might it make sense to include that in the constant names, e.g. `PAYPAL_ORDER_META_STATUS`? We don't have any other types of meta at the moment, but it also helps with future usage and expectations to have that aspect in the name.
```suggestion
	const PAYPAL_ORDER_META_STATUS = '_paypal_status';
```